pubsub/topic.go

```diff
@@ -592,6 +592,189 @@ func (i *IngestionDataSourceCloudStoragePubSubAvroFormat) isCloudStorageIngestio
 	return true
 }
 
+// EventHubsState denotes the possible states for ingestion from Event Hubs.
+type EventHubsState int
```

Comment:
Can we use AzureEventHubs to match the proto definition more closely?

Reply:
I shortened this (and other State consts) just because it was getting a bit lengthy and customers are only ever going to be checking for this output anyway, rather than setting it. Also considering we aren't supporting this surface (the v2 will be using a different admin surface), I felt being brief was ok.

I don't feel that strongly about it, so if you think it's best to match proto definition, I'm ok with that.

Reply:
I'm okay with keeping just EventHubsState then.